CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 - Fix systemd collector D-Bus connection. https://github.com/giantswarm/kubernetes-node-exporter/pull/44
 
+### Added
+
+- Added tolerations `GiantSwarmObservability=true` and `NodeWithImparedVolumes=true`.
```

Comment:
This shouldn't go into the same version unless you override git tags. See links at the bottom.